clang/include/clang/Format/Format.h

```diff
@@ -922,8 +922,23 @@ struct FormatStyle {
     ///   };
     ///   int f();
     ///   int f() { return 1; }
+    ///   int foooooooooooooooooooooooooooooooooooooooooooooooo::
+    ///       baaaaaaaaaaaaaaaaaaaaar();
     /// \endcode
     RTBS_None,
+    /// Break after return type automatically, while allowing a break after
+    /// short return types.
+    /// ``PenaltyReturnTypeOnItsOwnLine`` is taken into account.
+    /// \code
+    ///   class A {
+    ///     int f() { return 0; };
+    ///   };
+    ///   int f();
+    ///   int f() { return 1; }
+    ///   int
+    ///   foooooooooooooooooooooooooooooooooooooooooooooooo::baaaaaaaaaaaaaaaaaaaaar();
+    /// \endcode
+    RTBS_AllowShortType,
     /// Always break after the return type.
     /// \code
     ///   class A {
```

Comment:
Shouldn't it be added in this block also?

Reply:
Yes, it should indeed be added.